packages/manager/apps/telecom/src/app/telecom/pack/slots/informations/informations.constants.js

```diff
@@ -0,0 +1,9 @@
+export const IG_MAP_URL = ' https://www.ovhtelecom.fr/xdsl/incident/#/';
```

Comment:
```suggestion
export const IG_MAP_URL = 'https://www.ovhtelecom.fr/xdsl/incident/#/';
```
Also shouldn't we declare the sub or a `default` entry in case we have different urls in the future 
```suggestion
export const IG_MAP_URL = {
FR: 'https://www.ovhtelecom.fr/xdsl/incident/#/';
}
```